test/phoenix_live_view/html_engine_test.exs

```diff
@@ -1286,7 +1448,12 @@ defmodule Phoenix.LiveView.HTMLEngineTest do
     end
 
     test "invalid tag" do
-      message = ~r/.exs:1:(11:)? expected closing `}` for expression/
+      message = """
+      test/phoenix_live_view/html_engine_test.exs:1:11: expected closing `}` for expression
+        |
+      1 | <div foo={<%= @foo %>}>bar</div>
+        |            ^\
```

Comment:
I'm not sure if I got this error. What does it have todo with `for expression`? 🤔